package.json

```diff
@@ -44,25 +44,25 @@
     "treeview"
   ],
   "dependencies": {
-    "react-json-viewer-cool": "^1.0.9"
+    "react-json-viewer-cool": "alxblsk/react-json-viewer#major-deps-bump"
```

Comment:
this waits merging of https://github.com/5SSS/react-json-viewer/pull/17, to bump this dependency version to `2.0.0`.